6_visualize/src/prep_timeline_fun.R

```diff
@@ -0,0 +1,59 @@
+prep_timeline_fun <- function(DateTime, timestep_ind, timeline_config, spark_config, legend_text_cfg) {
+
+  this_DateTime <- as.POSIXct(DateTime, tz = "UTC") # WARNING, IF WE EVER MOVE FROM UTC elsewhere, this will be fragile/bad.
+
+  # Compute the full x limits for all datetimes
+  storm_timesteps <- fetch_read(timestep_ind)
```

Comment:
seems like a lot of reading (and fetching?) this tiny file when an arg to `timeline_fun_a_20180912_06` could be `date_lims = I(c('2018-01-01','2018-01-03'))`